indico/modules/events/abstracts/templates/abstract.html

```diff
@@ -10,9 +10,13 @@
         <div class="i-timetable-connect-down"></div>
         <div class="i-timeline-separator"></div>
         {{ render_abstract_comments(abstract) }}
-        {{ render_abstract_reviews(abstract) }}
-        <div class="i-timeline-separator"></div>
-        <div class="i-timetable-connect-up"></div>
-        {{ render_decision_box(abstract, form) }}
+        {% if abstract.is_convener(session.user) or abstract.can_judge(session.user) %}
+            {{ render_abstract_reviews(abstract) }}
+            <div class="i-timeline-separator"></div>
+        {% endif %}
+        {% if abstract.state.name == 'submitted' and abstract.can_judge(session.user) %}
```

Comment:
What about moving the state check into `can_judge` (and maybe `can_review` too)? We do the same in `can_edit`.

I'm quite sure we'll end up the ability to re-judge without resetting first at some time (we don't check it on the server at all right now :warning:) if we need to check both all the time.